### PR TITLE
[Bug] Prevent double click on tips; pass uuid4 transaction_id

### PR DESCRIPTION
**Pull request by yattias**

### CODE REVIEW

File: config/fetch.js [hunk 1]
```diff
@@ -326,6 +327,7 @@ export const supportContent = async ({ contentType, objectId, amount }) => {
     content_type: contentType,
     object_id: objectId,
     amount,
+    transaction_id: uuidv4(),
```
Comment (koutst) [inline config/fetch.js:330]: Would this generate a unique id on each submit? Shouldn't we generate it once on modal load?

**Review by koutst — APPROVED** [review]
LGTM

File: config/fetch.js (hunk 1 above)
Comment (yattias) [reply config/fetch.js:330]: You know, you are indeed right. On modal load is safer. Curious - do you have any plans when it comes to what to do with this field in the backend?

Comment (koutst) [reply config/fetch.js]: Just store it and query the db to check if the same client ID has already been used and fail if it has.

**Pull request merged**